chpl/chpl-service/src/main/java/gov/healthit/chpl/dao/impl/DeveloperDAOImpl.java

```diff
@@ -754,4 +762,18 @@ private DeveloperStatusEntity getStatusByName(final String statusName) {
         }
         return statuses.get(0);
     }
+
+    private List<DeveloperEntity> getEntitiesByCertificationBodyId(final List<Long> certificationBodyIds) {
+        String hql = "SELECT DISTINCT dev "
+                + "FROM CertifiedProductEntity cp "
+                + "JOIN FETCH cp.productVersion pv "
+                + "JOIN FETCH pv.product prod "
+                + "JOIN FETCH prod.developer dev "
+                + "WHERE cp.certificationBody.id IN :certificationBodyIds";
```

Comment:
I think I usually put parenthesis around the parameters in the "IN" clause, like "IN (:certificationBodyIds)" but I'm not sure if they are actually necessary or not. It seems like something hibernate might handle for you if they are left out. If you know this HQL runs then don't bother, but just wanted to note it in case.